app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/viewModel/StatsViewModel.kt

```diff
@@ -17,7 +20,42 @@ class StatsViewModel @Inject constructor(application: android.app.Application) :
     }
 
     @Inject
-    lateinit var model: String
+    lateinit var stats: Stats
 
-    fun getStats(): String = model
+    val mutableHp: MutableLiveData<Int> = MutableLiveData()
+    val mutableXp: MutableLiveData<Int> = MutableLiveData()
+
+    fun currentHp(): LiveData<Int> {
+        if (mutableHp.value == null) {
+            mutableHp.value = stats.hp
+        }
+        return mutableHp
+    }
+
+    fun increaseHp() {
+        stats.hp += 1
+        mutableHp.value = stats.hp
```

Comment:
seems a bit weird to me that you have to keep track of changes in two different places, the stats model and the live data 🤔  this is how it's supposed to work? :D (I have no idea about this stuff)

Reply:
me neither, but it should be doable somehow